tools/c7n_gcp/tests/test_dns.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018-2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gcp_common import BaseTest
+
+
+class DnsManagedZoneTest(BaseTest):
+
+    def test_managed_zone_query(self):
+        project_id = 'cloud-custodian'
+        session_factory = self.replay_flight_data(
+            'managed-zone-query', project_id=project_id)
+
+        policy = self.load_policy(
+            {'name': 'gcp-dns-managed-zone-dryrun',
+             'resource': 'gcp.dns-managed-zone'},
+            session_factory=session_factory)
+
+        managed_zone_resources = policy.run()
+        self.assertEqual(len(managed_zone_resources), 1)
```

Comment:
its always good to assert an attribute on the resource, to ensure your actually finding the thing you expect as well as that its shape/form is per expectation.

Reply:
Thank you for your suggestion, it makes sense, fixed it with the latest push.